### PR TITLE
Use builder for TextureDescriptor

### PR DESCRIPTION
**Pull request by cwfitzgerald**
This isn't meant to be merged as is, but this demos one usage of the new builders.

#478 #470 https://github.com/gfx-rs/wgpu/pull/840

### CODE REVIEW

File: examples/cube/main.rs [hunk 1]
```diff
@@ -173,15 +173,17 @@ impl framework::Example for Example {
...
-            usage: wgpu::TextureUsage::SAMPLED | wgpu::TextureUsage::COPY_DST,
-        });
+        let texture = device.create_texture(
+            wgpu::TextureDescriptor::new(
+                texture_extent,
+                wgpu::TextureDimension::D2,
+                wgpu::TextureFormat::Rgba8UnormSrgb,
+                wgpu::TextureUsage::SAMPLED | wgpu::TextureUsage::COPY_DST,
+            )
+            .label("label")
```
Comment (kvark) [inline examples/cube/main.rs:183]: Is rustfmt really formatting it this way?

Comment (cwfitzgerald) [reply examples/cube/main.rs:183]: Yeah. Would you've formatted it differently?

Comment (kvark) [reply examples/cube/main.rs:183]: Typically if you chain calls, they get aligned with indentation relative to the init line, i.e.
```rust
something
  .do_something()
  .and_something()
```

In our case, it looks like there is no indentation, so I'm surprised

Comment (multun) [reply examples/cube/main.rs:183]: that's the expected behavior inside parenthesized expressions I think

**Comment by cwfitzgerald** [comment]
Closing as it has served its purpose.

**Pull request closed without merging**